dev/mgr.go

```diff
@@ -227,7 +227,16 @@ func (m *Manager) NewConfiguration(ids []uint32, qspec QuorumSpec) (*Configurati
 	}
 
 	var cnodes []*Node
+	unique := make(map[uint32]struct{})
```

Comment:
Minor, but if you want you can use

```go
duplicate := make(map[uint32]bool)
```

as the set, and

```go
if unique[nid] {
```

but then perharps use another name for `unique`.